kiva/trait_defs/ui/wx/kiva_font_editor.py

```diff
@@ -51,16 +51,20 @@ def to_wx_font(self, editor):
         import kiva.constants as kc
 
         font = editor.value
-        weight = (wx.NORMAL, wx.BOLD)[font.weight == kc.BOLD]
-        style = (wx.NORMAL, wx.ITALIC)[font.style == kc.ITALIC]
+        weight = (
+            wx.FONTSTYLE_NORMAL, wx.FONTWEIGHT_BOLD
+        )[font.weight == kc.BOLD]
```

Comment:
It took me a second to understand why a `True`/`False` was being used to index into a tuple. 

I was going to ask that we rewrite this to be more readable - but given that this is the wx toolkit, let's not.

Reply:
Yeah... very strange